tests/tests_cogniswitch/test_cogniswitchtoolspec.py

```diff
@@ -0,0 +1,60 @@
+import requests
+from unittest.mock import Mock
+from llama_hub.tools.cogniswitch.base import CogniswitchToolSpec
```

Comment:
Since you have declared the init file, import can be done via 

`from llama_hub.tools.cogniswitch import CogniswitchToolSpec`